agents/chart/main.py

```diff
@@ -2,23 +2,38 @@
 
 import fixieai
 
-BASE_PROMPT = """I am an agent that makes sick charts."""
+BASE_PROMPT = """I am an agent that makes beautiful charts using the Quickchart API."""
 
 FEW_SHOTS = """
 Q: Make a chart of the 5 tallest mountains
-Thought: I need to make a JSON object with information about the 5 tallest mountains.
+Thought: I need to ask the router agent for the 5 tallest mountains and their elevations.
+Ask Agent[router]: What are the 5 tallest mountains and their elevations?
```

Comment:
I don't think this is what we want though as the LLM has all this in its internal memory already, there's no point in asking router.

Reply:
The problem then is that Chart always hallucinates. I believe we want chart to be a leaf agent and to ask external agents for the data to graph, and sometimes that will come from the LLM but most of the time it won't ... it might come from Twitter or GitHub or a Google Doc or whatever.

Reply:
That seems upside down though. As you say, chart can receive unformatted data from elsewhere. It shouldn't try to find it itself, especially when that is not likely to succeed.

Reply:
The problem is that Router doesn't know this. In other words, it sees that Chart claims to make Charts and hence when someone asks for a chart of <x>, it gets sent to the chart agent first ... forcing it to hallucinate the data.

Until we have a better approach to breaking down multi-step queries, and having Fixie and Router understand how to do this properly, having the Chart agent ask for the data it needs serves as a beautiful example of how to compose agents -- an Agent can ask another Agent for help. Chart should not try to be clever and hallucinate fake data; it is not a data provider; it is just a data visualizer. 

This is something I need for doing demos as well -- I need to show the idea that agents can talk to other agents and this is a fantastic way of doing that.

Reply:
Do you have an example playground where this actually works? It doesn't make sense to me.

Reply:
Yes: https://app.fixie.ai/sessions/agreeable-alike-baritone

I don't know why you'd stand in the way here. I'm constantly doing demos for people to show them this capability of our system, and Chart's current behavior is broken because it always hallucinates. I guess I could make a separate "betterchart" agent, but that seems silly. Explain the objection to making charts great again?